src/engraving/layout/ilayout.h

```diff
@@ -46,12 +46,15 @@ class FiguredBassItem;
 
 class Image;
 
+class EngravingItem;
 class LedgerLine;
 class SLine;
 class LineSegment;
 class Lyrics;
 
 class TextBase;
+class FiguredBassItem;
+enum class KerningType;
```

Comment:
Here the classes are added alphabetically and even grouped to make it easier. It would be good to keep with it.